### ISSUE TITLE
Empty index values creating empty links (Copyright Renewals)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1763]
Empty facet value at the top of the Author facet:
https://exhibits.stanford.edu/copyrightrenewals

It's possible for a keyboard or SR user to trigger this link:
https://exhibits.stanford.edu/copyrightrenewals/catalog?f%5Bauthor_person_facet%5D%5B%5D=

Empty titles at top of search results - 
https://exhibits.stanford.edu/copyrightrenewals/catalog?utf8=%E2%9C%93&exhibit_id=copyrightrenewals&search_field=search&q=

again, keyboard/SR can trigger:
https://exhibits.stanford.edu/copyrightrenewals/catalog/R458196

I'm not sure how this should be fixed - data remediation? If the metadata is accurate (can it be??) then the app needs to handle empty values in a way that makes them perceivable. 

These haven't been reported for any other exhibit, so hopefully it's not a broader problem. 

Tagging @caaster for attention.

**Comment by cbeer** [comment]
I think these blank values will go away with a reindex.

**Comment by mejackreed** [comment]
@caaster is it ok if I go ahead and reindex?

**Comment by cbeer** [comment]
Don't bother with copyrightrenewals; I'll be doing something to it for #1759

**Comment by caaster** [comment]
@hannahfrost question from Cory: he would like to reindex this exhibit to see if that fixes the problem. Is it ok for his to reindex?

**Comment by hannahfrost** [comment]
Yes, fine to reindex. Thanks!

—
Hannah Frost | Associate Director for Digital Services
Stanford Libraries<https://library.stanford.edu/> | Digital Library Systems and Services
Lathrop Library 384
E: ***@***.*** | C: (650) 799-1352
Pronouns: she/her

On Jul 23, 2024, at 7:57 AM, Cathy Aster ***@***.***> wrote:



@hannahfrost<https://github.com/hannahfrost> question from Cory: he would like to reindex this exhibit to see if that fixes the problem. Is it ok for him to reindex?

**Comment by corylown** [comment]
I am going to need more information about where the records for this collection come from. They have no corresponding resource in the database that I can find and therefore cannot be reindexed.

**Comment by dnoneill** [comment]
See https://stanford.enterprise.slack.com/archives/C0BTKKS8P/p1728919347213459?thread_ts=1728679512.575229&cid=C0BTKKS8P for original data
